lib/comm_common.cpp

```diff
@@ -164,8 +165,10 @@ static bool intranode_enabled[2][4] = { {false,false,false,false},
 static bool peer2peer_present = false;
 
 /** by default enable both copy engines and load/store access */
-static int enable_peer_to_peer = 3; 
+static int enable_peer_to_peer = 3;
 
+/** sets whether we cap which peers can use peer-to-peer */
```

Comment:
why is this mulled in here? Guess we need to handle #1024 first then?

Reply:
^ @mathiaswagner yes